portieris.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: portieris
+  version: 0.13.19
+  epoch: 0
+  description: A Kubernetes Admission Controller for verifying image trust.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  environment:
+    CGO_ENABLED: "0"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/IBM/portieris.git
+      tag: v${{package.version}}
+      expected-commit: d0705039d060ab11b8fa598b79647ac0008af57e
+
+  - uses: go/build
+    with:
+      packages: "./cmd/portieris"
+      ldflags: "-X github.com/IBM/portieris/internal/info.Version=v${{package.version}}"
+      tags: containers_image_openpgp
+      output: portieris
+
+subpackages:
+  - name: portieris-compat
+    description: Compat package for portieris
+    pipeline:
+      - runs: |
+          ln -s /usr/bin/portieris "${{targets.subpkgdir}}"/
```

Comment:
```suggestion
          mkdir -p ${{targets.subpkgdir}}/
          ln -s /usr/bin/portieris ${{targets.subpkgdir}}/portieris
```

Reply:
oh man thanks.  I was thinking "surely / already exists".

Reply:
running a test locally, then I will push.  Thanks for coming along for the ride

Reply:
subpackage in this case bundle things from `/home/build/melange-out/portieris-compat` and that doesn't exist out of the box. 
So what above command is doing is following: 

```bash
mkdir -p /home/build/melange-out/portieris-compat/
ln -s /usr/bin/portieris /home/build/melange-out/portieris-compat/portieris
```
You can put `exit 1` at places and run `make debug/foo` for debugging these cases.